src/index.test.ts

```diff
@@ -68,6 +65,7 @@ describe('rateLimit', () => {
       // Call finish before reaching the limit of 3
       await limiter.finish()
     } catch (e) {
+      console.log(e)
```

Comment:
Is this log intentional?

Reply:
Yes

Reply:
I didn't expect to see this within the context of a test.
Wouldn't `expect(e).toBe('rejectedPromise')` log the error value, if this failed during a test?

Reply:
For example, I'm imagining the test would log:
expect('500 - I made a whoops').toBe('rejectedPromise')

Which should (respectfully) make the log redundant? Or?

Reply:
Feedback addressed. Thanks.